uint8 agent_number
std_msgs/UInt8 state
